joomla/joomla-cms
# [4.0] user registration accessibility

The description strings were removed some time ago but they were not removed from the xml. This resulted in the fields being rendered with an aria-described being created that pointed to a non existent element.

### To test
enable user registration in com_users in the admin and then go to the user registration form in the front end

View the source of any of the inputs and you will see code like
![image](https://user-images.githubusercontent.com/1296369/89122275-e12dbd00-d4bd-11ea-9273-799ce2fe9fc3.png)

Apply the pr and view the generated source again and the same input will look like
![image](https://user-images.githubusercontent.com/1296369/89122282-f4d92380-d4bd-11ea-9289-bcce1a199d6b.png)
